action.yml

```diff
@@ -0,0 +1,72 @@
+inputs:
+  fetch-targets:
+    description: 'depviz target to fetch (format: \"org1/repo1 org1/repo2 org2 repo1\")'
+    default: '' 
+  generation-targets:
+    description: 'depviz target to generate outputs (format: \"org1/repo1 org1/repo2 org2 repo1\")' 
+    default: $GITHUB_REPOSITORY
+  branch: 
+    description: 'the branch where to push the generated files.' 
+    default: 'depviz-generate'
```

Comment:
```suggestion
```

I suggest we remove this completely, else you'll have multiple publish if you `uses` multiple time this action in the same workflow.

Just add a suggested snippet example instead of making it a built-in feature.